src/modules/settings/twitch/Settings.jsx

```diff
@@ -13,12 +14,21 @@ let mountedNode;
 
 export default class SettingsModule {
   constructor() {
-    this.renderSettings();
+    this.load();
     domObserver.on('a[data-test-selector="user-menu-dropdown__settings-link"]', () => {
       this.renderSettingsMenuOption();
     });
   }
 
+  async load() {
+    // eslint-disable-next-line import/no-unresolved
+    await import('../components/settings/global/*.jsx');
+    // eslint-disable-next-line import/no-unresolved
+    await import('../components/settings/twitch/*.jsx');
```

Comment:
should these be loaded by `'../components/Settings.jsx'`?

Reply:
https://github.com/night/betterttv/pull/5111#discussion_r789347202